src/models.ts

```diff
@@ -125,7 +190,7 @@ export interface ModelAndWeightsConfig {
  * @returns A `Promise` of `Model`, with the topology and weights loaded.
  */
 // TODO(cais): Add link to the core's documentation of `WeightManifestConfig`.
-export async function loadModelInternal(modelConfigPath: string):
+export async function loadModelFromPath(modelConfigPath: string):
```

Comment:
Looking at my earlier API comment, I think we're missing something here to support the scheme prefixed string sugaring.  Or is this an interim solution that's accepting raw strings without scheme information?